### ISSUE TITLE
[BUG] Timestamp error on Binance

### ISSUE THREAD

**Issue by Nullably** [CoinAlpha/hummingbot#1262]
**Describe the bug**
HB keeps on issuing errors after starting pure market making strategy on Binance (Windows 10). This happens to me occasionally when my system clock is out of sync with the Binance timestamp, I have to run a script to update system clock to rectify the problem. 
 
In last section of the log file:
```
2020-01-01 20:16:53,682 - 34628 - hummingbot.market.binance.binance_market - NETWORK - Unexpected error while fetching trading rules.
Traceback (most recent call last):
  File "hummingbot\market\binance\binance_market.pyx", line 753, in hummingbot.market.binance.binance_market.BinanceMarket._trading_rules_polling_loop
  File "hummingbot\core\utils\async_utils.py", line 19, in safe_gather
  File "hummingbot\market\binance\binance_market.pyx", line 330, in _update_trade_fees
  File "hummingbot\market\binance\binance_market.pyx", line 320, in hummingbot.market.binance.binance_market.BinanceMarket._update_trade_fees
  File "hummingbot\market\binance\binance_market.pyx", line 278, in query_api
  File "hummingbot\core\utils\async_call_scheduler.py", line 128, in call_async
  File "hummingbot\core\utils\async_call_scheduler.py", line 117, in schedule_async_call
  File "hummingbot\core\utils\async_call_scheduler.py", line 80, in _coro_scheduler
  File "concurrent\futures\thread.py", line 56, in run
  File "site-packages\binance\client.py", line 1795, in get_trade_fee
binance.exceptions.BinanceWithdrawException: BinanceWithdrawException: Timestamp for this request is outside of the recvWindow.
2020-01-01 20:16:53,683 - 34628 - hummingbot.market.binance.binance_market - WARNING - Could not fetch new trading rules from Binance. Check network connection.
```

Here are the steps to reproduce the issue (see attachments in the section below):
1. Manually set system clock of the machine to 1 minute behind (on Windows system).
2. Configure the bot to trade on Binance on pure market making strategy.
3. Start the bot

**Screenshots**
![image](https://user-images.githubusercontent.com/37262506/71642753-310b9200-2ceb-11ea-8635-db10c10acd79.png)

**Release version**
0.21.0
This also happens to development branch

**Optional: jack | coinalpha:**

**Comment by Hashira-Lars** [comment]
Tested with pr #1274 confirm fixes the issue

**Comment by martinkou** [comment]
Fixed with #1274.
